packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -0,0 +1,574 @@
+import dayjs from 'dayjs'
```

Comment:
Non blocking, but for later, we might need to see where we can consolidate all tx history helpers - right now they're present in `helpers.ts`, and `utils.ts`, and then some specific one's in `withdrawals > helpers` and `deposits > helpers`.

Reply:
Yeah we should really clean those up right after